docs/src/overview/build-deploy.md

```diff
@@ -1,118 +1,80 @@
 
-After you push your code, the first build step is to validate your configuration files
-(`.platform.app.yaml`, `.platform/services.yaml`, and `.platform/routes.yaml`).
-The Git server issues an error if validation fails, and nothing happens on the server.
+Install dependencies
+: If you have specified additional dependencies required by your build hook such as SaaS, webpack, or Drupal Console, and have provided valid versions, the dependencies get downloaded during this step.
 
-{{< note >}}
+Run build flavor commands
+: The build runs a series of standard commands based on the build flavor specified in the configuration file.
+It's possible to override this behavior and skipping the step.
 
-While most projects have a single `.platform.app.yaml` file,
-Platform.sh supports multiple applications in a single project.
-It scans the repository for `.platform.app.yaml` files in subdirectories
-and each directory containing one is built as an independent application.
-A built application doesn't contain any directories above the one in which it's found.
-The system is smart enough not to rebuild applications that have already been built,
-so if you have multiple applications, only changed applications are rebuilt and redeployed.
+Run build hook
+: The build hook comprises one or more shell commands that you write to finish creating your production code base.
+It could be compiling Sass files, running a bundler, rearranging files on disk or compiling an application in a compiled language.The committed build hook runs in the build container. During this time, commands have write access to the file system, but services (including other applications in your cluster) aren't yet available.
 
-{{< /note >}}
+Freeze app container
+: Once the previous steps are completed, the file system gets freezed and produce a read-only container image, which is the final build artifact. An artifact is a custom-built snapshot of your application that's reliable and repeatable.
+The container configuration for both your application and its underlying services is exclusively based on your configuration files.
+A container has a 1:1 relationship with a Git commit.
+This means that builds are repeatable and that if the system detects that there aren't changes for a container,
+it reuses the existing container image skipping the build step and saving time.
 
-The live environment is composed of multiple containers&mdash;both for your applications and for the services it depends on.
-It also has a virtual network connecting them as well as a router to route incoming requests to the appropriate application.
 
-Based on your application type, the system selects one of the pre-built container images and runs the following:
+## The deploy
+The deploy process makes those containers live, replacing the previous versions, with virtually no interruption in service.
```

Comment:
It's unclear to me what "those" containers are here. This is a new section and multiple containers weren't just talked about.